### DISCUSSION TITLE
Development server

### DISCUSSION THREAD

**Discussion by Liberate30** [resend/react-email#712] [Q&A]
I followed all the instructions in the docs but when I run `npm run dev` the cmd creates a folder called **.react-email-temp** and then throws the error below.

> Error: EEXIST: file already exists, mkdir '.react-email-temp'
>     at Object.mkdirSync (node:fs:1396:3)
>     at downloadClient (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\react-email\dist\source\utils\download-client.js:19:18)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async Command.dev (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\react-email\dist\source\commands\dev.js:20:9) {
>   errno: -4075,
>   syscall: 'mkdir',
>   code: 'EEXIST',
>   path: '.react-email-temp'
> }

**Comment by bukinoshita** [comment]
Can you try deleting the `.react-email-temp` and re-running that command?

**Reply by Liberate30** [reply]
I generated a folder called `.react-email-temp`
Then inside it I have a file called `react-email.tar.gz` file

Then I saw the following error 

> Error: ENOENT: no such file or directory, lstat '.react-email-temp\client'
>     at Object.lstatSync (node:fs:1593:3)
>     at Object.lstatSync (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\graceful-fs\polyfills.js:318:34)
>     at statFunc (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\fs-extra\lib\util\stat.js:24:20)
>     at getStatsSync (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\fs-extra\lib\util\stat.js:25:19)
>     at Object.checkPathsSync (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\fs-extra\lib\util\stat.js:67:33)
>     at Object.moveSync (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\fs-extra\lib\move\move-sync.js:14:52)
>     at downloadClient (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\react-email\dist\source\utils\download-client.js:23:24)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async Command.dev (C:\Users\COREI5\Desktop\My works\react-email-starter\node_modules\react-email\dist\source\commands\dev.js:20:9) {
>   errno: -4058,
>   syscall: 'lstat',
>   code: 'ENOENT',
>   path: '.react-email-temp\\client'
> }

**Comment by Liberate30** [comment]
From the videos I watched on youtube when the tutor ran `npm run dev` the code generated a folder and some other folders like email and files inside it. But mine developed a compressed file instead.
